lighthouse-core/report/html/report-styles.css

```diff
@@ -66,6 +66,7 @@
   --lh-audit-group-vpadding: 8px;
   --lh-section-vpadding: 12px;
   --chevron-size: 12px;
+  --bytes-col-width: 3px;
```

Comment:
this is super confusing to look at on it's own, 0px didn't do the trick right? maybe comment to explain :)